.golangci.yml

```diff
@@ -11,6 +11,8 @@ linters:
     - typecheck         # See golangci/golangci-lint#419
     - unused            # deprecated
     - varcheck          # unreliable
+    - gocyclo           # gives errors unrelated to diff
+    - interfacer        # gives errors unrelated to diff
 
```

Comment:
I am not particularly comfortable disabling these. @cshari-zededa the idea here is NOT that you're only responsible for literally just the lines that your patch touches, but that we all have collective responsibility to *gradually* fix yetus issues around things that our patches touch.

I know it could be painful at times, but I think it is for the better in general for EVE.

Reply:
@rvs Our observation for gocyclo was that if you change one line in a 1000 line function you are expected to rewrite that whole function to reduce its complexity measure. That doesn't make any sense. I have no issue with enforcing this for brand new functions which are added.

I don't recall what the issue was with the interfacer check. @cshari-zededa ?

Reply:
interfacer check throws errors for unrelated method, which was not touched at all. It is unreliable.

Reply:
@cshari-zededa can you please provide an example of that?

Reply:
@nordmark fair enough, but I like a suggestion that we can keep complexity decreasing. Would love to hear from @aw-was-here how easy it would be to implement something like this in yetus

Reply:
@rvs here is the example for interfacer error (error #2 there):
https://310-182198941-gh.circle-artifacts.com/0/tmp/yetus-out/diff-patch-golangcilint.txt

And here is the linter's official github page, which says this particular linter is deprecated and is prone to bad suggestions:
https://github.com/mvdan/interfacer

Reply:
Ok. Disabling Interfacer seems reasonable then!